packages/app/src/game/module/chat/sagas.ts

```diff
@@ -1,15 +1,16 @@
 import { take, delay, select, put } from "redux-saga/effects";
 import { quickChatCommands } from ".";
+import { PlayerActions, PlayerEvents } from "../../../../../gamemode/lib";
 
 
 export const handleQuickChat = function*() {
 	while (true) {
-		const action = yield take(quickChatCommands.setPlayerChat.toString());
+		const action = yield take(PlayerEvents.playerReceiveQuickChatEvent.toString());
 		const { sendingPlayerId, receivingPlayerId, chatValue } = action.payload;
 		if (!chatValue) {
 			return;
 		}
-
+		yield put(quickChatCommands.setPlayerChat({ sendingPlayerId, receivingPlayerId, chatValue }));
 		let chatToCheck = chatValue;
 		while (true) {
```

Comment:
This is currently blocking new quick chats until previous quick chat has been cleared - need to find a fix

Reply:
Resolving as copy of suggestion below